cBinder/WrapperBuilder.py

```diff
@@ -245,22 +250,19 @@ def _build_python_wrapper_for_function(self, writer, module_name, function):
 
             parameter_list = []
             for param in function.parameters:
+                str1 = param.name + unique_identifier_suffix
                 if param.is_out and 'char' in param.type and not param.is_pointer_to_array:
-                    parameter_list.append(
-                            param.name + unique_identifier_suffix + '[0].encode() if ' +
-                            'type(' + param.name + unique_identifier_suffix + '[0]) is str ' +
-                            'else ' + param.name + unique_identifier_suffix + '[0]\n\t\t\t')
+                    str2 = '[0].encode() if ' + 'type(' + param.name + unique_identifier_suffix + '[0]) is str '
```

Comment:
additional variables are being used (like br{1,2,3} below, too) instead of concatenating multiple strings in multiple lines. It's questionable whether it's better to have additional variables with meaningless names here.